商品一覧表示

fujiyasu0522 opened the pull request:
# WHAT
商品一覧表示機能を実装

# WHY
商品の一覧をトップページに表示するため

GIF
https://gyazo.com/4981e32156bfb53568e2fe72d2cca85c

fujiyasu0522 committed:
商品一覧表示

https://gyazo.com/4981e32156bfb53568e2fe72d2cca85c

strongmasahiro commented:
レビュー依頼ありがとうございます！
こちらのプルリクエストですが、実際の画面や機能ができているか確認させていただきたいので、下記キャプチャ動画をご提出いただけますでしょうか？

```
・商品のデータがない場合に、ダミー商品が表示されている動画
```

その後、再度レビュー依頼フォームにてご提出をお願いいたします。

fujiyasu0522 committed:
ダミー画像の修正

商品のデータがない時のダミー画像
https://gyazo.com/bf7fbf7dbdcc3774b05e6038bd66d54d

商品データがあるときのダミー画像
https://gyazo.com/8a25832fa3986269f5b7b8e96381a50b

fujiyasu0522 commented:
商品のデータがない時のダミー画像
https://gyazo.com/bf7fbf7dbdcc3774b05e6038bd66d54d

商品データがあるときのダミー画像
https://gyazo.com/8a25832fa3986269f5b7b8e96381a50b

strongmasahiro commented on app/views/items/index.html.erb:
```diff
@@ -123,28 +123,26 @@
-        <%= link_to "#" do %>
+    <% if @items.present? %>
+      <% @items.each do |item| %>
+        <%= link_to item_path(item.id) do %>
```
`<%= link_to item_path(item.id) do %>`
こちらのパスに関しては、今後の実装で記載する箇所になりますので以下の記述に戻しましょう！
```
<%= link_to "#" do %>
```

strongmasahiro commented on app/views/items/index.html.erb:
```diff
@@ -123,28 +123,26 @@
-      <li class='list'>
-        <%= link_to "#" do %>
+    <% if @items.present? %>
+      <% @items.each do |item| %>
```
`<li class='list'>`
こちらの記述が消えてしまっています。 `<% @items.each do |item| %>`と`<%= link_to "#" do %>`の間に追加しましょう！

strongmasahiro commented on app/views/items/show.html.erb:
```diff
@@ -4,7 +4,7 @@
   <div class="item-box">
     <h2 class="name">
-      <%= "商品名" %>
+      <%= item.item_name %>
```
`<%= item.item_name %>`
こちらの記述は今後の実装になるためコメントアウトしておきましょう！

strongmasahiro commented on app/views/items/show.html.erb:
```diff
@@ -19,7 +19,7 @@
       </span>
       <span class="item-postage">
-        <%= "配送料負担" %>
+        <%= item.delivery_cost_id %>
```
` <%= item.delivery_cost_id %>`
こちらの記述は今後の実装になるためコメントアウトしておきましょう！

strongmasahiro commented on app/views/items/show.html.erb:
```diff
@@ -43,7 +43,7 @@
         <tr>
           <th class="detail-item">出品者</th>
-          <td class="detail-value"><%= "出品者名" %></td>
+          <td class="detail-value"><%= @user.name %></td>
```
`<%= @user.name %>`
こちらの記述は今後の実装になるためコメントアウトしておきましょう！

strongmasahiro reviewed (commented):
レビュー依頼ありがとうございます！
修正点確認させていただきました。
追加でコメントいたしましたので、ご確認お願いします！

fujiyasu0522 committed:
修正1

chobi555 commented:
レビュー依頼ありがとうございます！
LGTMとさせていただきます、実装お疲れさまでした！

マージ後、ローカルのmasterブランチに忘れずにプルしましょう。
その後、新しいブランチを作成し次の実装に取り組んでいただけますと幸いです。
引き続き頑張っていきましょう！
